heartbeat/monitors/active/icmp/icmp.go

```diff
@@ -89,29 +90,30 @@ func (jf *jobFactory) checkConfig() error {
 	return nil
 }
 
-func (jf *jobFactory) makeJobs() (j []jobs.Job, endpoints int, err error) {
+func (jf *jobFactory) makePlugin() (plugin2 plugin.Plugin, err error) {
```

Comment:
Why the names returned parameters? Seems that the code always defines the return in the statement of the function.